### DISCUSSION TITLE
Applist ordering

### DISCUSSION THREAD

**Discussion by reloxx13** [LizardByte/community#489] [Sunshine] [answered]
The applist is not ordered. Ordering by app/game name would be a small but useful fix for users.

Workaround
Currently, i paste the applist json content in chatgpt and tell it to order the list by appname for me.

**Comment by ReenigneArcher** [comment]
I agree, this needs to be added.

**Reply by ReenigneArcher** [reply, marked as answer]
See: https://github.com/LizardByte/Sunshine/pull/3223

**Reply by ReenigneArcher** [reply]
This is merged.
